services/slam/server_test.go

```diff
@@ -143,6 +146,24 @@ func TestWorkingServer(t *testing.T) {
 		test.That(t, mockServer.rawBytes, test.ShouldResemble, internalStateSucc)
 	})
 
+	t.Run("working GetLatestMapInfo", func(t *testing.T) {
+		someProtoTime := timestamppb.Now()
```

Comment:
[q] can we replace var name with either t or timestamp?

Reply:
Yeah I'll clean this up